apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/SubscriberClientImpl.SingleChannel.cs

```diff
@@ -310,7 +313,8 @@ private void StartStreamingPull()
         // Backoff delay (if present) has already been done; no need to delay here.
         private void HandleStartStreamingPullWithoutBackoff()
         {
-            _pull = _client.StreamingPull(CallSettings.FromCancellationToken(_softStopCts.Token));
+            var callSettings = CallSettings.FromHeader(XGoogRequestParamsHeaderKey, string.Format("subscription={0}", _subscriptionName));
```

Comment:
This should use `Uri.EscapeDataString` on the subscription name - or we can wait until there's a new GAX release, hopefully soon, and then use `CallSettings FromGoogleRequestParamsHeader`. (We can refactor to that later easily enough.)

Reply:
Sorry, that was left over from a previous change that isn't meant to be here. It is now removed.